src/Type/Php/ExplodeFunctionDynamicReturnTypeExtension.php

```diff
@@ -57,10 +58,17 @@ public function getTypeFromFunctionCall(
 		}
 
 		$stringType = $scope->getType($args[1]->value);
+		$accessory = [];
 		if ($stringType->isLowercaseString()->yes()) {
-			$returnValueType = new IntersectionType([new StringType(), new AccessoryLowercaseStringType()]);
+		    $accessory[] = new AccessoryLowercaseStringType();
+		}
+		if ($stringType->isUppercaseString()->yes()) {
+		    $accessory[] = new AccessoryUppercaseStringType();
+		}
+		if (count($accessory) > 0) {
+		    $returnValueType = new IntersectionType([new StringType(), ...$accessory]);
```

Comment:
```
$accessoryTypes[] = new StringType();
$returnValueType = new IntersectionType($accessoryTypes);
```
will maybe avoid lint error on lower php versions.

Reply:
Applied in 4c54a1a44

Reply:
Linting issues resolved.